tests/distributions/test_mvn.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2017-2019 Uber Technologies, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+
+from pyro.distributions import MultivariateNormal
+from tests.common import assert_equal
+
+
+def random_mvn(loc_shape, cov_shape, dim):
+    """
+    Generate a random MultivariateNormal distribution for testing.
+    """
+    rank = dim + dim
+    loc = torch.randn(loc_shape + (dim,))
```

Comment:
nit: why not `.requires_grad_()` here?

Reply:
That should be fine too. I'll do it instead. :)